manim/scene/scene_file_writer.py

```diff
@@ -499,13 +523,20 @@ def open_partial_movie_stream(self, file_path=None):
             self.video_container = video_container
             self.video_stream = stream
 
+            self.queue = Queue()
```

Comment:
```suggestion
            self.queue: Queue[tuple[float, PixelArray]] = Queue()
```
Maybe? (`PixelArray` is from `manim.typing`)